quinn-h3/src/qpack/decoder.rs

```diff
@@ -0,0 +1,623 @@
+use bytes::{Buf, BufMut};
+use std::io::Cursor;
+
+use super::static_::{Error as StaticError, StaticTable};
+use super::vas;
+use super::{
+    DynamicTable, DynamicTableDecoder, DynamicTableError, DynamicTableInserter, HeaderField,
+};
+
+use super::block::{
+    HeaderBlocField, HeaderPrefix, Indexed, IndexedWithPostBase, Literal, LiteralWithNameRef,
+    LiteralWithPostBaseNameRef,
+};
+use super::stream::{
+    Duplicate, DynamicTableSizeUpdate, EncoderInstruction, HeaderAck, InsertCountIncrement,
+    InsertWithNameRef, InsertWithoutNameRef, StreamCancel,
+};
+use super::ParseError;
+
+use super::prefix_int;
+use super::prefix_string;
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
```

Comment:
If this error (and the others like it) ever reach a public API, it'd be nice to derive `Display` and `Error` impls for them using err-derive (see e.g. `quinn_proto::ConnectionError` for an example)